ChatQnA/benchmark/performance/helm_charts/customize.yaml

```diff
@@ -6,7 +6,7 @@ HUGGINGFACEHUB_API_TOKEN: ${HF_TOKEN}
 podSpecs:
   - name: chatqna-backend-server-deploy
     spec:
-      image_name: opea/chatqna-no-wrapper
+      image_name: opea/chatqna
```

Comment:
Those change under `benchmark` shouldn't include in your PR, please fix the rebasing.